app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -54,24 +54,22 @@ const NoteForm = ({ onClose, refetch, note, isEdit }) => {
               name="description"
             />
             <Select
-              defaultMenuIsClose
               isSearchable
               required
               className="w-full flex-grow-0"
               label="Assigned Contacts"
-              name="assignedContacts"
+              name="assignedContact"
               options={CONTACTS_DROPDOWN_DATA}
-              placeholder="Select Role"
+              placeholder="Select contact"
             />
             <Select
-              defaultMenuIsClose
               isMulti
               required
               className="w-full flex-grow-0"
               label="Tags"
               name="tags"
               options={TAGS_DROPDOWN_DATA}
-              placeholder="Select Role"
+              placeholder="Select Tags"
```

Comment:
Maintain some consistency. Stick onto Sentence case for the messages shown in UI. Your placeholders follow different casing right now.